nbs/models.tsmixerx.ipynb

```diff
@@ -515,6 +515,7 @@
     "#| hide\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #67.    def inflate_df(df_in, multiplier=513):
</pre><p>I think it's easier and clearer to use <code>neuralforecast.utils.generate_series</code> here, since that allows you to set the number of series to create.</p><p> </p><p>Also this notebook takes 50 seconds to test (<a href="https://github.com/Nixtla/neuralforecast/actions/runs/8628136142/job/23649556872?pr=962#step:5:97" target="_blank">logs</a>) so can you please set <code>max_steps=5</code>  to reduce the time?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/962/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
Thanks, good points. Changed it.